### PR TITLE
[mavlink_messages] Missmatch between header macros and class used

### PR DESCRIPTION
**Pull request by loicdubois**
### Solved Problem
Trying to get the SYSTEM_TIME mavlink stream to work 
```
WARN  [mavlink] stream SYSTEM_TIME not found
ERROR [mavlink] configure_streams_to_default() failed
```
Fixes #{Github issue ID}

### Solution
Simply add the stream corresponding to the right header preprocessor directives

### Changelog Entry
Bugfix: SYSTEM_TIME stream not found

### Alternatives
-

### Test coverage
- add stream to desired mode
- SITL starts without complaining

### Context
-

### CODE REVIEW

**Comment by junwoo091400** [comment]
Sorry, accidentally closed it!

**Comment by dagar** [comment]
Good catch @loicdubois, perhaps we should just generate this instead.

File: src/modules/mavlink/mavlink_messages.cpp [hunk 1]
```diff
@@ -323,8 +323,11 @@ static const StreamListItem streams_list[] = {
 #if defined(COMMAND_LONG_HPP)
 	create_stream_list_item<MavlinkStreamCommandLong>(),
 #endif // COMMAND_LONG_HPP
-#if defined(SYSTEM_TIME_HPP)
+#if defined(SYS_STATUS_HPP)
 	create_stream_list_item<MavlinkStreamSysStatus>(),
+#endif // SYS_STATUS_HPP
+#if defined(SYSTEM_TIME_HPP)
```
Comment (dagar) [inline src/modules/mavlink/mavlink_messages.cpp:329]: SYSTEM_TIME is now being added twice (see below ~ line 379).

**Review by dagar — CHANGES_REQUESTED** [review]
Fixing SYS_STATUS is correct, but SYSTEM_TIME doesn't need to be added.

**Comment by loicdubois** [comment]
> What is the use case for this? Could you give some context?

I actually don't use it, just got a warning by playing around with streams to add and remove

**Review by dagar — APPROVED** [review]

**Pull request merged**